### PR TITLE
672 cyril style forgot password page

### PR DESCRIPTION
**Pull request by cyrielo**
# Description
Create react component for forgot password page and apply styling

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix a feature that knowingly causes existing functionality to not work as expected)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?

- [ ] Login Screen
- [ ] Single Upload Page
- [ ] Batch Upload Page
- [ ] All Projects Page
- [ ] Sample Details Page
- [ ] Report Page

Specific components 

* 

# Testing Script:

###Login Page
* Successfully login to the application and get taken to the project page

###Single Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Uploading a duplicate file throws an error
* Pipeline gets initiated successfully 

###Batch Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Navigate to batch upload settings page after upload 
* Host selection from previous page still selected 
* Uploading a duplicate file throws a warning
* Pipeline gets initiated successfully 

###All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

###Sample Details Page
* Information is consistent from Project Page
* Notes are editable and save successfully 
* Can navigate to report page 

###Report Page
* Data loads succesfully 
* Collapse and Uncollapse button works
* Clicking on the taxonomy name gives you total reads for that taxon
* Category filter works
* Genus Search works
* Filters work as expected
* Disabling Filters works
* Filtering by column works 

# Checklist:

- [ ] I have run through the testing script to make sure current functionality is unchanged
- [ ] I have done relevant tests that prove my fix is effective or that my feature works
- [ ] I have spent time testing out edge cases for my feature
- [ ] I have updated the test script or pull request template if necessary
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/assets/src/components/Header.jsx [hunk 1]
```diff
@@ -56,31 +56,34 @@ class Header extends React.Component  {
...
-          <p>1. Install and configure the Amazon Web Services Command Line Interface (AWS CLI).</p>
-          <p>2. Install the IDseq CLI:</p>
-          <p><span className="code">pip install git+https://github.com/chanzuckerberg/idseq-cli.git</span></p>
-          <p>3. Upload a sample using a command of the form:</p>
-          <div className="code center-code">
-            <p>idseq -p '<span className="code-to-edit">Your Project Name</span>' -s '<span className="code-to-edit">Your Sample Name</span>' \</p>
-            <p> -u https://idseq.net -e <span className="code-personal">{this.userDetails.email}</span> -t <span className="code-personal">{this.user_auth_token}</span> \</p>
-            <p> --r1 <span className="code-to-edit">your_sample_R1</span>.fastq.gz --r2 <span className="code-to-edit">your_sample_R2</span>.fastq.gz</p>
+    let cli_modal;
+    if (this.userSignedIn) {
```
Comment (yunfangjuan) [inline app/assets/src/components/Header.jsx]: why not just 
if (this.userSignedIn && this.demoUser !== 1) {
}
you don't need another ternary operator.

**Comment by cdebourcy** [comment]
LGTM, please rebase before merging, I just made a change to the CLI modal text
